src/Application/README.md

```
@@ -0,0 +1,12 @@
+# Application Layer
+
+Contains application services that orchestrate domain objects to fulfill use cases.
+- Commands: Write operations that change state
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)